### DISCUSSION TITLE
[Feature] Add Search / Filtering on Shared Albums

### DISCUSSION THREAD

**Discussion by cfitzw** [immich-app/immich#12096] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would like the ability for album shares to be able to leverage the search capabilities, such as keywords and person selection.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2995

**Reply by cfitzw** [reply]
This is not the same. It appears that is for searching shared albums.

This is if I share an album (to a non-Immich user), they should be able to do some basic searches, like selecting faces, etc..

**Reply by tulku** [reply]
Indeed this is a different issue. This feature request is to add search / filtering options to the Album view. This can be very useful when sharing albums, particularly with people outside of the immich server. Filtering by camera type, dates, faces would make viewing large albums much better.

**Comment by Twisterado** [comment]
Would also really like this!
Especially filtering by "shared by" to only look at pictures uploaded by specific users (e.g. to hide my own pictures and look at what others have shared from the joint vacation).

Also: is there any possibility to sponsor a Feature Request?
